zerorobot/cli/server.py

```diff
@@ -69,6 +69,18 @@ def start(listen, data_repo, template_repo, config_repo, config_key, debug,
         robot.add_template_repo(url)
 
     robot.set_data_repo(data_repo)
+    if not config_repo:
+        namespace_in_cfg = j.core.state.configGetFromDict("myconfig", "namespace", "")
+        if namespace_in_cfg:
+            config_repo_path = j.sal.fs.joinPaths(j.dirs.HOMEDIR, "configmgrsandboxes", namespace_in_cfg) 
+            if j.sal.fs.exists(config_repo_path):
+                config_repo = config_repo_path
+
+    if not config_key:
```

Comment:
where does this `config_key` variable comes from ?

Reply:
config_key comes from a command argument parameter 
```python
@click.option('--config-key', '-K', required=False, help='Absolute path to ssh key to secure configuration data, which is committed and pushed (see auto-push) in the configuration repo.\
```
specified for the start function.